modules/brainbrowser/ajax/minc.php

```diff
@@ -22,13 +22,59 @@
 
 $headers = array();
 
-$query     = "select File from files where FileID = :MincID";
-$minc_file = $DB->pselectOne($query, array('MincID' => $_REQUEST['minc_id']));
-$minc_file = getMincLocation() . $minc_file;
+if (strpos($_REQUEST['minc_id'], 'l') !== false) {
```

Comment:
I'm not sure that this will work as intended. If you look [here](https://www.w3schools.com/php/php_operators.asp), `!==` will always return `true` when supplied with numeric values. I tried it on the command line with -1, 0, and 1 and this was the case.

As `strpos` always returns a number, this condition will always be true.